sql/moz-fx-data-shared-prod/subscription_platform_derived/stripe_subscriptions_history_v1/query.sql

```diff
@@ -234,6 +236,14 @@ customers AS (
     AND UPPER(
       LEFT(customers.shipping_address_postal_code, 1)
     ) = ca_shipping_postal_districts.postal_district_code
+  LEFT JOIN
```

Comment:
I added this for completeness but there aren't billing addresses that have postal code but no state which I'm guessing is why it wasn't already here